enclave-tls/src/Makefile

```diff
@@ -23,35 +23,39 @@ endif
 libenclave_tls := $(notdir $(Enclave_Tls_Lib))
 crypto_wrappers := $(addsuffix .so,$(addprefix libcrypto_wrapper_,nullcrypto wolfcrypt))
 tls_wrappers := $(addsuffix .so,$(addprefix libtls_wrapper_,nulltls wolfssl))
-enclave_quotes := $(addsuffix .so,$(addprefix libenclave_quote_,nullquote))
+attesters := $(addsuffix .so,$(addprefix libattester_,nullattester))
+verifiers := $(addsuffix .so,$(addprefix libverifier_,nullverifier))
 
 ifdef OCCLUM
   Enclave_Tls_Cflags += -DOCCLUM
-  enclave_quotes += $(addsuffix .so,$(addprefix libenclave_quote_,sgx-ecdsa))
+  attesters += $(addsuffix .so,$(addprefix libattester_,sgx-ecdsa))
+  verifiers += $(addsuffix .so,$(addprefix libverifier_,sgx-ecdsa))
 endif
 
 ifdef SGX
   crypto_wrappers += $(addsuffix .so,$(addprefix libcrypto_wrapper_,wolfcrypt-sgx))
   tls_wrappers += $(addsuffix .so,$(addprefix libtls_wrapper_,wolfssl-sgx))
-  enclave_quotes += $(addsuffix .so,$(addprefix libenclave_quote_,sgx-ecdsa sgx-ecdsa-qve sgx-la))
+  attesters += $(addsuffix .so,$(addprefix libattester_,sgx-ecdsa sgx-la))
+  verifiers += $(addsuffix .so,$(addprefix libverifier_,sgx-ecdsa sgx-ecdsa-qve sgx-la))
 endif
 
-all: $(addprefix $(Build_Libdir)/,$(libenclave_tls) $(crypto_wrappers) $(tls_wrappers) $(enclave_quotes))
+all: $(addprefix $(Build_Libdir)/,$(libenclave_tls) $(crypto_wrappers) $(tls_wrappers) $(attesters) $(verifiers))
 
 this_dir := $(shell pwd)
 libenclave_tls_files := \
   $(wildcard \
     $(this_dir)/core/*.c $(this_dir)/api/*.c \
     $(this_dir)/crypto_wrappers/internal/*.c $(this_dir)/crypto_wrappers/api/*.c \
     $(this_dir)/tls_wrappers/internal/*.c $(this_dir)/tls_wrappers/api/*.c \
-    $(this_dir)/enclave_quotes/internal/*.c $(this_dir)/enclave_quotes/api/*.c \
+    $(this_dir)/attesters/internal/*.c $(this_dir)/attesters/api/*.c \
+    $(this_dir)/verifiers/internal/*.c $(this_dir)/verifiers/api/*.c \
     $(this_dir)/util/*.c \
   )
 
 ocall_files := \
   $(this_dir)/sgx/ocalls.c \
   $(this_dir)/tls_wrappers/wolfssl-sgx/ocalls.c \
-  $(this_dir)/enclave_quotes/sgx-ecdsa/ocalls.c
+  $(this_dir)/attesters/sgx-ecdsa/ocalls.c
```

Comment:
Verifiers don't have ocalls?

Reply:
others are not used. But I will add them into here.